workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/HandleCreatePlayerRequestSystem.cs

```diff
@@ -13,19 +15,33 @@ public class HandleCreatePlayerRequestSystem : ComponentSystem
         private struct CreatePlayerData
         {
             public readonly int Length;
+
             [ReadOnly] public ComponentDataArray<PlayerCreator.CommandRequests.CreatePlayer> CreatePlayerRequests;
-            public ComponentDataArray<PlayerCreator.CommandResponders.CreatePlayer> CreatePlayerResponders;
+
             public ComponentDataArray<WorldCommands.CreateEntity.CommandSender> CreateEntitySender;
         }
 
         [Inject] private CreatePlayerData createPlayerData;
 
+        private struct EntityCreationResponses
+        {
+            public readonly int Length;
+            public ComponentDataArray<PlayerCreator.CommandResponders.CreatePlayer> CreatePlayerResponders;
+            public ComponentDataArray<WorldCommands.CreateEntity.CommandResponses> CreateEntityResponses;
+        }
+
+        [Inject] private EntityCreationResponses entityCreationResponseData;
+
+        private class EntityCreationRequestContext
```

Comment:
Maybe rename to `PlayerCreationRequestContext` to make more specific? Especially so since we filter out other create entity responses that aren't player creation

Reply:
aye